Library/Homebrew/utils.rb

```diff
@@ -19,6 +19,7 @@
 require "utils/svn"
 require "utils/tty"
 require "tap_constants"
+require "PATH"
```

Comment:
Why is this needed?

Reply:
This is needed in order for `PATH.new` in `which` method to work. `which` is used by `os_version` method in `os/linux.rb`

Reply:
Gotcha, thanks!